app/src/main/java/com/example/henzoshimada/feeltrip/FeelTripApplication.java

```diff
@@ -59,4 +65,79 @@ public static Participant getParticipant(){
         return participant;
     }
 
+    private static FilterController filterController = null;
+    public static FilterController getFilterController(){
+        if (filterController == null){
+            filterController = new FilterController();
+        }
+        return filterController;
```

Comment:
For the purposes of grabbing the FilterController no matter where we are in the app.